desktop_version/src/Render.cpp

```diff
@@ -1644,6 +1651,11 @@ void gamerender(void)
         }
     }
 
+    if (graphics.fademode==0 && !game.intimetrial && !game.isingamecompletescreen && game.swngame != 1 && game.showingametimer) {
```

Comment:
Opening braces should have their own line.

Reply:
new line added